v3/src/components/case-table/case-table-types.ts

```diff
@@ -21,7 +21,10 @@ export interface TCalculatedColumn extends CalculatedColumn<TRow> {}
 export interface TRenderers extends Renderers<TRow, unknown> {}
 export interface TRenderEditCellProps extends RenderEditCellProps<TRow> {}
 export interface TRenderCellProps extends RenderCellProps<TRow> {}
-export interface TRenderHeaderCellProps extends RenderHeaderCellProps<TRow> {}
+export interface TRenderHeaderCellProps extends RenderHeaderCellProps<TRow> {
+  attrIdToEdit?: string
+  setAttrIdToEdit?: (attrId: string | undefined) => void
+}
```

Comment:
My inclination would be to store this information in the `CollectionTableModel` rather than in React state, in which case this change wouldn't be required.